webpack/assets/javascripts/react_app/redux/actions/statistics/statistics.test.js

```diff
@@ -33,7 +28,7 @@ describe('statistics actions', () => {
 
     store.dispatch(actions.getStatisticsData(failedRequestData));
     await IntegrationTestHelper.flushAllPromises();
-    expect(store.getActions()).toEqual(onFailureActions);
+    expect(store.getActions()).toMatchSnapshot();
```

Comment:
How is changing the statistics test related to this PR? :)

Reply:
I remember we talked about it in several other API related PRs,
that we should update snapshots instead of storing it in a fixture, though in the end we didn't do it.

I find myself fighting each time with the fixtures file which does exactly what snapshot does,
but it is a lot harder to update it compared to a snapshot.